### ISSUE TITLE
About Training data

### ISSUE THREAD

**Issue by zs1314** [zjunlp/OceanGPT#8]
Hello! I would like to ask if you guys plan to publish the pre-training data and seed corpus, thank you!

**Comment by zxlzr** [comment]
hi, sorry,  due to application requirements, we are currently unable to make the data public. However, we plan to release a portion of the data after the next version update.

**Comment by zs1314** [comment]
> 您好，很抱歉，由于应用程序要求，我们目前无法公开数据。但是，我们计划在下一次版本更新后发布部分数据。

OK.Thank you

**Comment by zs1314** [comment]
> 部分指令数据可在 https://huggingface.co/datasets/zjunlp/OceanInstruct

I don't understand, is this the data used for reasoning?

**Comment by zxlzr** [comment]
> > 部分指令数据可在 https://huggingface.co/datasets/zjunlp/OceanInstruct
> 
> I don't understand, is this the data used for reasoning?

This is part of the data for  instruction tuning.
